MPSGameLift/Documentation/GameLift.md

```diff
@@ -97,7 +67,7 @@ It's important to make sure that the bootstrap.game.profile.setreg file has been
     ---
     **NOTE**
 
-    Note: launch_server.cfg is required because there's a bug with multiplayer when calling --loadlevel in the command-line. See https://github.com/o3de/o3de/issues/15773.
+    Launch_server.cfg is required because there's a bug with multiplayer when calling --loadlevel in the command-line. See https://github.com/o3de/o3de/issues/15773.
```

Comment:
like with .setreg files, you can validate to confirm the file exists as you expect it.

Reply:
Definitely! I'll add the checks in a future PR when I actually create the "GameLiftServerPackageWindows" folder.